inc/container.class.php

```diff
@@ -620,7 +620,7 @@ public function prepareInputForAdd($input)
             }
         }
 
-        $input['itemtypes'] = Sanitizer::dbEscape(json_encode($input['itemtypes']));
+        $input['itemtypes'] = json_encode($input['itemtypes']);
 
         return $input;
     }
```

Comment:
dedicated tab should have an icon (same as fields plugin igon)

![image](https://github.com/user-attachments/assets/b8403136-e3f4-45df-8c4f-f2a82201185e)